io.github.tntwise.REAL-Video-Enhancer.yml

```diff
@@ -0,0 +1,43 @@
+app-id: io.github.tntwise.REAL-Video-Enhancer
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: REAL-Video-Enhancer
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --socket=wayland
+  - --device=dri
+  - --share=network
+  - --filesystem=xdg-videos
+  - --filesystem=/var/lib/flatpak/app:ro
```

Comment:
why do you need this for?

Reply:
reading out the permissions in the file located at /var/lib/flatpak/app/io.github.tntwise.REAL-Video-Enhancer/x86_64/master/active/metadata

Reply:
> reading out the permissions in the file located at /var/lib/flatpak/app/io.github.tntwise.REAL-Video-Enhancer/x86_64/master/active/metadata

Why?

You can read `/.flatpak-info` instead inside the sandbox.

Reply:
Oh, thank you.  I will change that.